File: src/main/java/idiotamspielen/vttproject/classifications/CharacterClass.java
package idiotamspielen.vttproject.classifications;

import java.util.List;

public class CharacterClass extends Things {
    private String name;
    private String description;
    private int hitPoints;
    private int armorClass;


    public CharacterClass(String name, String description, List classFeatures) {
        this.name = name;
        this.description = description;
    }

    public String getName() {
        return name;
    }
}

File: src/main/kotlin/idiotamspielen/vttproject/classifications/CharacterClass.kt
package idiotamspielen.vttproject.classifications

open class CharacterClass(name1: String?, description1: String?, abilities1: MutableList<*>?) : Nameable{
    private val name: String? = null
    private val description: String? = null
    private val abilities: List<Ability>? = null

    override fun getName(): String {
        return name!!
    }
}
